sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -961,7 +969,24 @@ object ScalaJSPluginInternal {
         new PhantomJettyClassLoader(jettyLoader, getClass.getClassLoader)
       },
       scalaJSJavaSystemProperties := Map.empty,
-      scalaJSConfigurationLibs := Nil
+      scalaJSConfigurationLibs := Nil,
+      scalaJSInstallJsdom := {
+        val jsdomDirectory = target.value / "scalajs-jsdom"
+        val logger = streams.value.log
+        if (!jsdomDirectory.exists()) {
+          val npm = sys.props("os.name").toLowerCase match {
+            case os if os.contains("win") ⇒ "cmd /c npm"
+            case _ ⇒ "npm"
+          }
+          IO.createDirectory(jsdomDirectory)
+          val process = Process(s"$npm install jsdom", jsdomDirectory)
```

Comment:
We should not install a completely unknown version like that. That undermines the default stability of a Scala.js build. 

We should have a setting for the version of `jsdom` that should be installed instead.